src/lib/routes/util.ts

```diff
@@ -29,14 +29,10 @@ export const handleErrors: (
     error: Error,
 ) => void = (res, logger, error) => {
     if (createError.isHttpError(error)) {
-        return (
-            res
-                // @ts-expect-error http errors all have statuses, but there are no
-                // types provided
-                .status(error.status ?? 400)
-                .json({ message: error.message })
-                .end()
-        );
+        return res
+            .status(error.status ?? 400)
+            .json({ message: error.message })
+            .end();
```

Comment:
we don't need .end() to close the write stream since json() will do that.

Reply:
heh, fair shout, I can update it, but it wasn't actually related to this PR. tsc just pointed out that we no longer got a tsc-error here, so had to remove the @ts-expect-error comment

Reply:
And I see we still got that error, because the build failed, it was just the ts-expect-error had ended up in the wrong place. Updated to remove the end() call and readd the ts-expect-error with an explanation in 9499986